hbp_knowledge/receptors/zoli2018.bel

```diff
@@ -83,214 +92,337 @@ a(MESH:Acetylcholine) -- bp(MESH:Apoptosis)
 p(MESH:"Receptors, Nicotinic") -- path(MESH:Anxiety)
 p(MESH:"Receptors, Nicotinic") -- path(MESH:Depression)
 p(MESH:"Receptors, Nicotinic") -- path(MESH:"Alzheimer Disease")
 p(MESH:"Receptors, Nicotinic") -- path(MESH:"Parkinson Disease")
 p(MESH:"Receptors, Nicotinic") -- path(MESH:Pain)
 p(MESH:"Receptors, Nicotinic") -- path(MESH:Epilepsy)
+UNSET Confidence
 
 SET Evidence ="nAChRs are particularly important in two critical periods of brain life: early pre- and post-natal circuit formation, and age-related cell degeneration. They are involved in neuronal survival, as it has been shown that nicotinic agonists are neu- roprotective in in vivo and in vitro models"
+SET Confidence = "Medium"
 p(MESH:"Receptors, Nicotinic") -- a(MESH:Neurons)
-a(MESH:Nicotine) -| a(MESH:Neurons)
+a(MESH:Nicotine) -> a(MESH:Neurons)
+#evidence says nicotinic agonists being neuro roprotective
+UNSET Confidence
 
 SET Evidence ="Once in the bloodstream, nicotine, rapidly crosses the blood/brain barrier, and accumulates and exerts its pharmacological effects [9, 58] (including psy- chostimulation, reward and the reduction of stress and anxi- ety) in the brain by binding to nAChRs."
+SET Confidence = "High"
 complex(a(MESH:Nicotine),p(MESH:"Receptors, Nicotinic"),loc(MESH:Brain)) -| path(MESH:Anxiety)
+UNSET Confidence
 
 SET Evidence ="Chronic nicotine exposure induces neural adaptations that change cell physi- ology and behaviour mainly as a result of activation and/or desensitisation of nAChRs. Studies of the brains of animals and smokers chronically exposed to nicotine have shown an increase in the number of nAChRs (up-regulation)."
-a(MESH:Nicotine) -| p(MESH:"Receptors, Nicotinic")
+SET Confidence = "Medium"
+a(MESH:Nicotine) -> p(MESH:"Receptors, Nicotinic")
+#in the last line of evidence it is suggested that nicotine upregulates nAChRs. In the line above it is suggested that nicotine leads to activation and/or desensitisation.
+UNSET Confidence
 
 SET Evidence ="The up- regulation of nAChRs has also been obtained using nicotinic agonists (cytisine, carbamylcholine and varenicline) [66, 67], antagonists (dihydro-β-erythroidine, mecamylamine) [68-70] and a partial agonist (CC4) "
-a(HBP:cytisine) -| p(MESH:"Receptors, Nicotinic")
-a(MESH:Carbachol) -| p(MESH:"Receptors, Nicotinic")
-a(MESH:Varenicline) -| p(MESH:"Receptors, Nicotinic")
-a(MESH:"Dihydro-beta-Erythroidine") -| p(MESH:"Receptors, Nicotinic")
-a(MESH:Mecamylamine) -| p(MESH:"Receptors, Nicotinic")
-a(HBP:cytisine) -| act(a(MESH:Nicotine))
-a(MESH:Carbachol) -| act(a(MESH:Nicotine))
-a(MESH:Varenicline) -| act(a(MESH:Nicotine))
+SET Confidence = "Medium"
+a(HBP:cytisine) -> p(MESH:"Receptors, Nicotinic")
+a(MESH:Carbachol) -> p(MESH:"Receptors, Nicotinic")
+a(MESH:Varenicline) -> p(MESH:"Receptors, Nicotinic")
+a(MESH:"Dihydro-beta-Erythroidine") -> p(MESH:"Receptors, Nicotinic")
+a(MESH:Mecamylamine) -> p(MESH:"Receptors, Nicotinic")
+a(HBP:cytisine) -> act(a(MESH:Nicotine))
```

Comment:
I would stick with the evidences for receptor upregulation but I couldn't find any information about nicotine activity upregulation, so I would not add these statements